MSAL/src/public/MSALPublicClientApplication.h

```diff
@@ -31,6 +31,7 @@
 @class MSALResult;
 @class MSALAccount;
 @class MSALTokenRequest;
+@class MSALAuthority;
 
 @interface MSALPublicClientApplication : NSObject
```

Comment:
getAccounts API will also need to be updated to be asynchronous for multiple authority aliases, but let's take it separately.